bin/pull-from-crunchy.sh

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 REG_CCP_IMAGE_PREFIX=registry.crunchydata.com/crunchydata
-for CONTAINER in crunchy-prometheus crunchy-upgrade crunchy-grafana crunchy-collect crunchy-pgbadger crunchy-pgpool crunchy-backup crunchy-postgres crunchy-postgres-gis crunchy-pgbouncer crunchy-pgadmin4 crunchy-pgdump crunchy-pgrestore crunchy-backrest-restore crunchy-scheduler crunchy-pgbench crunchy-pgbasebackup-restore
+for CONTAINER in crunchy-prometheus crunchy-upgrade crunchy-grafana crunchy-collect crunchy-pgbadger crunchy-pgpool crunchy-backup crunchy-postgres crunchy-postgres-gis crunchy-pgbouncer crunchy-pgadmin4 crunchy-pgdump crunchy-pgrestore crunchy-backrest-restore crunchy-scheduler crunchy-pgbench crunchy-pgbasebackup-restore crunchy-postgres-ha
```

Comment:
@andrewlecuyer `crunchy-postgres-gis-ha` too?

Reply:
`crunchy-postgres-gis-ha` is now included